tests/Database/DatabaseSchemaBlueprintIntegrationTest.php

```diff
@@ -284,11 +284,6 @@ public function testAddUniqueIndexWithoutNameWorks()
 
         $expected = [
             [
-                'CREATE TEMPORARY TABLE __temp__users AS SELECT name FROM users',
-                'DROP TABLE users',
-                'CREATE TABLE users (name VARCHAR(255) DEFAULT NULL COLLATE "BINARY")',
-                'INSERT INTO users (name) SELECT name FROM __temp__users',
-                'DROP TABLE __temp__users',
```

Comment:
It seems DBAL is now smarter here and doesn't creates temporary tables anymore when only a unique constraint or an index is added to the table.